packages/gitlab/src/components/widgets/ReleasesCard/ReleasesCard.tsx

```diff
@@ -145,11 +144,9 @@ export const ReleasesCard = (props: ReleasesCardProps) => {
         value?.releases
             .filter(makeFilter(show))
             .sort((a, b) => {
-                try {
-                    return rcompare(a.tag_name, b.tag_name);
-                } catch (error) {
-                    return a.tag_name > b.tag_name ? 1 : -1;
-                }
+                const d1 = new Date(a.released_at || a.created_at);
```

Comment:
If we change sorting, we must update the comment in line 142.

Reply:
I would like to add sort props to avoid breakchange, what do you think?

Reply:
LGTM!

Reply:
But by default, I would like to maintain the new sorting method.

Reply:
We could add a Select component for the sorting method.

Reply:
<img width="316" alt="image" src="https://github.com/immobiliare/backstage-plugin-gitlab/assets/18688410/fd73555c-541b-4143-b3fb-c0a9610d58e7">
<img width="281" alt="image" src="https://github.com/immobiliare/backstage-plugin-gitlab/assets/18688410/2bdd487c-7a9c-4372-adee-2ecb5d2fc674">

Selecting the sorting method is optional and is not shown by default.

Reply:
We can show it by default. WDYT?

Reply:
I just want to keep the same experience as before the modification. And the default sorting should be in line with what most users expect

Reply:
The feature slightly changes the user experience and improves the UI. We can maintain it active by default.

Reply:
Okay, agreed.

I've made changes and look forward to your review again!😀